cluster-up/cluster/kind/common.sh

```diff
@@ -145,26 +153,27 @@ function setup_kind() {
 
     _wait_kind_up
     _kubectl cluster-info
+    if [ $(nodes |wc -l) -ge 2 ]; then
```

Comment:
Do we have access to KUBEVIRT_NUM_NODES parameter in this context?

Reply:
I think so, but let's do this thing depending on the system.